source/_snippet.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="snippet">
+  <header class="snippet-header">
```

Comment:
Just played with this, TIL.

I'll drop it altogether for now, and just keep the `h3`. We may want to reintroduce a wrapping `div` if we introduce a "copy code" button or something down the road.